packages/engine-core/src/binary/BinaryEngine.ts

```diff
@@ -447,10 +447,24 @@ ${dim("In case we're mistaken, please report this to us 🙏.")}`)
       await this.stopPromise
     }
 
+    const retries = { times: 10 }
```

Comment:
Just curious, why 10?

Reply:
I ran it 350+ times a few times locally, that's what it took to remove the flakiness. Terrible logic, but is mostly for tests.

Reply:
Yes, we don't expect this to be as bad in production indeed.